saleor/dashboard/order/forms.py

```diff
@@ -248,14 +248,10 @@ def save(self, commit=True):
 class OrderNoteForm(forms.ModelForm):
     class Meta:
         model = OrderNote
-        fields = ['content', 'is_public']
+        fields = ['content']
         widgets = {
             'content': forms.Textarea()}
-        labels = {
-            'content': pgettext_lazy('Order note', 'Note'),
-            'is_public': pgettext_lazy(
-                'Allow customers to see note toggle',
-                'Customer can see this note')}
+        labels = {'content': pgettext_lazy('Order note', 'Note')}
 
     def send_confirmation_email(self):
```

Comment:
I think this should be removed as well, was used for notifying an user about new notes on his order